ambari-server/src/main/python/ambari-server.py

```diff
@@ -558,6 +558,7 @@ def init_ldap_setup_parser_options(parser):
   parser.add_option('--ldap-sync-username-collisions-behavior', default=None, help="Handling behavior for username collisions [convert/skip] for LDAP sync", dest="ldap_sync_username_collisions_behavior")
   parser.add_option('--ldap-force-lowercase-usernames', default=None, help="Declares whether to force the ldap user name to be lowercase or leave as-is", dest="ldap_force_lowercase_usernames")
   parser.add_option('--ldap-pagination-enabled', default=None, help="Determines whether results from LDAP are paginated when requested", dest="ldap_pagination_enabled")
+  parser.add_option('--ldap-enforcement', default=None, help="Forces the use of LDAP even if other (i.e. PAM) authentication method is configured already or if there is no authentication method configured at all", dest="ldap_enforcement")
```

Comment:
I think this should be a boolean, like `--silent`, which can be achieved by:

```
action="store_true", dest="ldap_enforcement", default=False, ...
```

Also, I find the name `enforcement` a bit odd.  Usually this kind of option is called `force` something.  I suggest renaming it to `--force-ldap-setup`, or `--ldap-force-setup` if you want to stick to the `ldap` prefix.

Reply:
Good idea; I'll change it.
Thanks!

Reply:
Fixed.